2022-03-30 13:06:14,090 INFO    MainThread:2906 [wandb_setup.py:_flush():69] setting env: {}
2022-03-30 13:06:14,090 INFO    MainThread:2906 [wandb_setup.py:_flush():69] setting login settings: {}
2022-03-30 13:06:14,091 INFO    MainThread:2906 [wandb_init.py:_log_setup():336] Logging user logs to /workspace/graph-neural-pde/src/wandb/run-20220330_130614-2fdfy1hb/logs/debug.log
2022-03-30 13:06:14,091 INFO    MainThread:2906 [wandb_init.py:_log_setup():337] Logging internal logs to /workspace/graph-neural-pde/src/wandb/run-20220330_130614-2fdfy1hb/logs/debug-internal.log
2022-03-30 13:06:14,092 INFO    MainThread:2906 [wandb_init.py:init():369] calling init triggers
2022-03-30 13:06:14,092 INFO    MainThread:2906 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2022-03-30 13:06:14,092 INFO    MainThread:2906 [wandb_init.py:init():418] starting backend
2022-03-30 13:06:14,092 INFO    MainThread:2906 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-30 13:06:14,137 INFO    MainThread:2906 [backend.py:ensure_launched():123] starting backend process...
2022-03-30 13:06:14,166 INFO    MainThread:2906 [backend.py:ensure_launched():128] started backend process with pid: 2956
2022-03-30 13:06:14,169 INFO    MainThread:2906 [wandb_init.py:init():423] backend started and connected
2022-03-30 13:06:14,171 INFO    MainThread:2906 [wandb_init.py:init():465] updated telemetry
2022-03-30 13:06:14,171 INFO    MainThread:2906 [wandb_init.py:init():484] communicating current version
2022-03-30 13:06:14,952 INFO    MainThread:2906 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-30 13:06:14,953 INFO    MainThread:2906 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2022-03-30 13:06:15,473 INFO    MainThread:2906 [wandb_init.py:init():522] starting run threads in backend
2022-03-30 13:06:19,067 INFO    MainThread:2906 [wandb_run.py:_console_start():1524] atexit reg
2022-03-30 13:06:19,067 INFO    MainThread:2906 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2022-03-30 13:06:19,068 INFO    MainThread:2906 [wandb_run.py:_redirect():1376] Redirecting console.
2022-03-30 13:06:19,069 INFO    MainThread:2906 [wandb_run.py:_redirect():1438] Redirects installed.
2022-03-30 13:06:19,069 INFO    MainThread:2906 [wandb_init.py:init():546] run started, returning control to user process
2022-03-30 13:06:19,210 INFO    MainThread:2906 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2022-03-30 13:06:19,212 INFO    MainThread:2906 [wandb_run.py:_restore():1466] restore
2022-03-30 13:06:21,764 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1177
  total_bytes: 1177
}

2022-03-30 13:06:22,508 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1177
  total_bytes: 1533
}

2022-03-30 13:06:23,046 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1177
  total_bytes: 7248
}

2022-03-30 13:06:23,150 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1177
  total_bytes: 7248
}

2022-03-30 13:06:23,254 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6894
  total_bytes: 7248
}

2022-03-30 13:06:23,357 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6894
  total_bytes: 7248
}

2022-03-30 13:06:23,461 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6894
  total_bytes: 7248
}

2022-03-30 13:06:23,564 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6936
  total_bytes: 7248
}

2022-03-30 13:06:23,668 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6936
  total_bytes: 7248
}

2022-03-30 13:06:23,771 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6936
  total_bytes: 7248
}

2022-03-30 13:06:23,878 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7248
  total_bytes: 7248
}

2022-03-30 13:06:23,981 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7248
  total_bytes: 7248
}

2022-03-30 13:06:24,085 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7248
  total_bytes: 7248
}

2022-03-30 13:06:24,188 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7248
  total_bytes: 7248
}

2022-03-30 13:06:24,292 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7248
  total_bytes: 7248
}

2022-03-30 13:06:24,395 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7248
  total_bytes: 7248
}

2022-03-30 13:06:24,499 INFO    MainThread:2906 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7248
  total_bytes: 7248
}

2022-03-30 13:06:25,805 INFO    MainThread:2906 [wandb_run.py:_show_files():1838] logging synced files
